tools/aws_lab_setup/README.md

```diff
@@ -1,50 +1,72 @@
-Ansible AWS training provisioner
-================================
+# Ansible AWS training provisioner
+**aws_lab_setup** is an automated lab setup for Ansible training on AWS (Amazon Web Services).  There are currently two modes:
+ - Ansible Essentials mode (default)
+ - Ansible Networking mode
 
-This is an automated lab setup for Ansible training. It creates four nodes per user in the `users` list.
+## Ansible Essentials Mode
+The default mode provisions four nodes per user in the `users` list.
 
 * One control node from which Ansible will be executed from and where Ansible Tower can be installed
 * Three web nodes that coincide with the three nodes in lightbulb's original design
 * And one node where `haproxy` is installed (via lightbulb lesson)
 
-**NOTE**: Because of [a bug introduced in Ansible v2.2.1](https://github.com/ansible/lightbulb/issues/112) you will need to run this provisioner with v2.3.2 or higher.
+## Ansible Networking Mode
+(**coming soon**)  
```

Comment:
That's a bit misleading - we have it now ;) How about ```**work in progress**``` if you want to highlight that it is still beta?

Reply:
Or just state `Ansible v2.4.2 or higher is required`

assuming this works with 2.4 and devel (soon to be 2.5)

Reply:
tried to fix this with more detail in commit: https://github.com/network-automation/lightbulb/commit/aa78f130ba60c0ac3f71e7a8b85fdb1cfc1969d9